cli/cliutils/stdout_logging.go

```diff
@@ -119,7 +120,7 @@ func LogLinux(instanceId string, tailing bool) {
 				if err != nil {
 					Fatal(NOT_FOUND, msgPrinter.Sprintf("%v could not be opened or does not exist: %v", sysLogPath, err))
 				}
-				defer file.Close()
+				defer cutil.CloseFileLogError(file)
```

Comment:
This `defer` is within a for loop. Can we not using defer because the scope of the defer is the whole function. It will be called at the end of the function exit in normal and error cases. Can we put `cutil.CloseFileLogError(file)` in these exit places instead?

Reply:
There's only one file open at a time here so this isn't necessary since there is a defer file.Close() before the loop starts. Removed this line